x-pack/plugins/cases/server/authorization/utils.test.ts

```diff
@@ -0,0 +1,293 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { nodeBuilder } from '../../../../../src/plugins/data/common';
+import { OWNER_FIELD } from '../../common';
+import {
+  combineFilterWithAuthorizationFilter,
+  ensureFieldIsSafeForQuery,
+  getOwnersFilter,
+  includeFieldsRequiredForAuthentication,
+} from './utils';
+
+describe('utils', () => {
+  describe('combineFilterWithAuthorizationFilter', () => {
+    it('returns undefined if neither a filter or authorizationFilter are passed', () => {
+      expect(combineFilterWithAuthorizationFilter()).toBeUndefined();
+    });
+
+    it('returns a single KueryNode when only a filter is passed in', () => {
+      const node = nodeBuilder.is('a', 'hello');
+      expect(combineFilterWithAuthorizationFilter(node)).toMatchInlineSnapshot(`
+        Object {
+          "arguments": Array [
+            Object {
+              "type": "literal",
+              "value": "a",
+            },
+            Object {
+              "type": "literal",
+              "value": "hello",
+            },
+            Object {
+              "type": "literal",
+              "value": false,
+            },
+          ],
+          "function": "is",
+          "type": "function",
+        }
+      `);
+    });
+
+    it('returns a single KueryNode when only an authorizationFilter is passed in', () => {
+      const node = nodeBuilder.is('a', 'hello');
+      expect(combineFilterWithAuthorizationFilter(undefined, node)).toMatchInlineSnapshot(`
+        Object {
+          "arguments": Array [
+            Object {
+              "type": "literal",
+              "value": "a",
+            },
+            Object {
+              "type": "literal",
+              "value": "hello",
+            },
+            Object {
+              "type": "literal",
+              "value": false,
+            },
+          ],
+          "function": "is",
+          "type": "function",
+        }
+      `);
+    });
+
+    it("returns a single KueryNode and'ing together the passed in parameters", () => {
```

Comment:
nit: and'ing ? 😄

Reply:
haha yeah, I'm just trying to convey that the action is the `and` so I hacked it into a verb 😆